### PR TITLE
Pull themes and plugins from web config

### PR DESCRIPTION
**Pull request by dkanada**

### CODE REVIEW

File: src/scripts/settings/webSettings.js [hunk 1]
```diff
@@ -18,11 +18,29 @@ function getDefaultConfig() {
     });
 }
 
-export function enableMultiServer() {
+export function getMultiServer() {
     return getConfig().then(config => {
         return config.multiserver;
     }).catch(error => {
-        console.log('cannot get web config:', error);
+        console.log('cannot get web config: ', error);
```
Comment (dkanada) [inline src/scripts/settings/webSettings.js:25]: Does JavaScript automatically insert the space here?

Comment (dmitrylyzo) [reply src/scripts/settings/webSettings.js:25]:
```
console.log('t:', window);
t: Window {parent: Window, opener: null, top: Window, length: 0, frames: Window, …}

console.log('t: ', window);
t:  Window {parent: Window, opener: null, top: Window, length: 0, frames: Window, …}
```

Comment (dkanada) [reply src/scripts/settings/webSettings.js:25]: Looks like it does, thanks.

File: src/scripts/autoThemes.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import * as userSettings from 'userSettings';
```
Comment (MrTimscampi) [inline src/scripts/autoThemes.js:1]: Not a fan of the name of this file. It could also probably be part of skinManager.

Comment (dkanada) [reply src/scripts/autoThemes.js:1]: I pulled this naming from `autoBackdrops` which I also hate. It was mainly to get a unirform name for those two weird files for now but we should probably remove them yeah.

**Pull request merged**